scrapers_library/data_portals/ckan/README.md

```diff
@@ -0,0 +1,215 @@
+      "organization": {
+          "id": "1149ee63-2fff-494e-82e5-9aace9d3b3bf",
+          "name": "city-of-new-york",
+          "title": "City of New York",
+          "description": "",
+          ...
+      },
+      "title": "NYPD Arrest Data (Year to Date)",
+      "extras": [
+          {
+              "key": "accessLevel",
+              "value": "public"
+          },
+          {
+              "key": "landingPage",
+              "value": "https://data.cityofnewyork.us/d/uip8-fykc"
+          },
+          {
+              "key": "publisher",
+              "value": "data.cityofnewyork.us"
+          },
+          ...
+      ],
+      "groups": [
+          {
+              "description": "Local Government Topic - for all datasets with state, local, county organizations",
+              "display_name": "Local Government",
+              "id": "7d625e66-9e91-4b47-badd-44ec6f16b62b",
+              "name": "local",
+              "title": "Local Government",
+              ...
+          }
+      ],
+      "resources": [
+          {
+              "created": "2020-11-10T17:05:37.001960",
+              "description": "",
+              "format": "CSV",
+              "id": "c48f1a1a-5efb-4266-9572-769ed1c9b472",
+              "metadata_modified": "2020-11-10T17:05:37.001960",
+              "name": "Comma Separated Values File",
+              "no_real_name": true,
+              "package_id": "f468fe8a-a319-464f-9374-f77128ffc9dc",
+              "url": "https://data.cityofnewyork.us/api/views/uip8-fykc/rows.csv?accessType=DOWNLOAD",
+              ...
+          },
+          {
+              "created": "2020-11-10T17:05:37.001970",
+              "describedBy": "https://data.cityofnewyork.us/api/views/uip8-fykc/columns.rdf",
+              "describedByType": "application/rdf+xml",
+              "description": "",
+              "format": "RDF",
+              "id": "5c137f71-4e20-49c5-bd45-a562952195fe",
+              "metadata_modified": "2020-11-10T17:05:37.001970",
+              "name": "RDF File",
+              "package_id": "f468fe8a-a319-464f-9374-f77128ffc9dc",
+              "url": "https://data.cityofnewyork.us/api/views/uip8-fykc/rows.rdf?accessType=DOWNLOAD",
+              ...
+          },
+          ...
+      ],
+      "tags": [
+          {
+              "display_name": "arrest",
+              "id": "a76dff3f-cba8-42b4-ab51-1aceb059d16f",
+              "name": "arrest",
+              "state": "active",
+              "vocabulary_id": null
+          },
+          {
+              "display_name": "crime",
+              "id": "df442823-c823-4890-8fca-805427bd8dd9",
+              "name": "crime",
+              "state": "active",
+              "vocabulary_id": null
+          },
+          ...
+      ],
+      "relationships_as_subject": [],
+      "relationships_as_object": [],
+      ...
+}
+```
+
+---
```

Comment:
For this part here, I'm curious if this is providing anything that's not already in the docstrings for these functions. 

Either way, I think it's best to migrate whatever is unique into the docstrings of the functions themselves and then either remove or condense to a higher level this section. 

Having this documentation in two places raises the risk that it will become misaligned, and if folks can easily access this logic in the docstrings themselves, there's not as much need to have it in the README.